website/docs/d/schema_site.html.markdown

```diff
@@ -15,7 +15,7 @@ Data source for MSO schema site, to fetch the MSO schema site details.
 ```hcl
 data "mso_schema_site" "sample_schema_site" {
```

Comment:
add empty line before and after like you did on the others.

Reply:
ok